pages/coinbase/signin.py

```diff
@@ -8,12 +8,35 @@
 
 class SigninPage:
 
-    def __init__(self, driver, drivethrough=True, bypass_auth=True):
+    def __init__(self, driver, spawn=False, bypass_auth=True):
+        """
+        inputs
+        -----
+        driver = selenium driver
+        spawn: bool
+            The default use case is that a driver that is already at the 
+            appropriate url is passed in as part of a user workflow. if 
+            spawn=True, the driver will be navigated to the appropriate url on 
+            instantiation.
+        bypass_auth: bool
+            Two-factor authentication is manditory. When bypass_auth is true,
+            cookies for the option to bypass are added before login. The cookie
+            file is in /secure and not tracked, so one must be built:
```

Comment:
"when bypass_auth is true, cookies to bypass that option are added before the login"